doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -592,7 +592,7 @@ to keep your client decoupled from implementation details.
 
 # Group Entities
 
-### List entities [GET /v2/entities{?limit,offset,options,type,id,idPattern,q,geometry,coords,attrs}]
+### List entities [GET /v2/entities{?limit,offset,options,type,id,idPattern,q,georel,geometry,coords,attrs}]
```

Comment:
It seems that georel and coords are not rendered in the parameters list at Apiary. I have tried to solve locally (with apiary client preview functionality), without luck :(